core/src/main/java/com/datastax/oss/driver/internal/core/metadata/SchemaAgreementChecker.java

```diff
@@ -108,7 +109,12 @@ private void sendQueries() {
     } else {
       CompletionStage<AdminResult> localQuery =
           query("SELECT schema_version FROM system.local WHERE key='local'");
-      CompletionStage<AdminResult> peersQuery = query("SELECT * FROM system.peers");
+
+      // `tokens` column is excluded, it is served from
+      // context.getMetadataManager().getMetadata().getTokenMap()
+      CompletionStage<AdminResult> peersQuery =
+          query(
+              "SELECT host_id, schema_version, rpc_address, native_address, native_port, data_center, rack FROM system.peers");
```

Comment:
Does Scylla have native_port and such? I thought only Cassandra?

Reply:
ok, now it is finally gone.